project/src/main/world/career-map.gd

```diff
@@ -112,6 +125,7 @@ func _random_levels() -> Array:
 	# mode over and over until they get the exact levels they want to play.
 	var seed_int := 0
 	seed_int += PlayerData.career.daily_earnings
+	seed_int += PlayerData.career.hours_passed
```

Comment:
Document this change in the commit comment

Reply:
Commit comment mentions `Random level seed now includes hours passed. This way if a player skips
or advances the clock without earning any money, they'll still get
different levels.`